apps/console/src/features/applications/components/settings/access-configuration.tsx

```diff
@@ -635,116 +699,291 @@ export const AccessConfiguration: FunctionComponent<AccessConfigurationPropsInte
+                                                                        ? SupportedAuthProtocolTypes.OIDC
+                                                                        : protocol
+                                                                ]
+                                                            }
+                                                            initialValues={
+                                                                get(
+                                                                    inboundProtocolConfig,
+                                                                    protocol ===
+                                                                                SupportedAuthProtocolTypes.OAUTH2_OIDC
+                                                                        ? SupportedAuthProtocolTypes.OIDC
+                                                                        : protocol
+                                                                )
+                                                                    ? inboundProtocolConfig[
+                                                                        protocol === SupportedAuthProtocolTypes
+                                                                            .OAUTH2_OIDC
+                                                                            ? SupportedAuthProtocolTypes.OIDC
+                                                                            : protocol
+                                                                    ]
+                                                                    : undefined
+                                                            }
+                                                            onSubmit={
+                                                                (values: any) => handleSubmit(values, protocol)
+                                                            }
+                                                            type={ protocol as SupportedAuthProtocolTypes }
+                                                            onApplicationRegenerate={
+                                                                handleApplicationRegenerate
+                                                            }
+                                                            onApplicationRevoke={ handleApplicationRevoke }
+                                                            readOnly={
+                                                                readOnly || !hasRequiredScopes(
+                                                                    featureConfig?.applications,
+                                                                    featureConfig?.applications?.scopes?.update,
+                                                                    allowedScopes
+                                                                )
```

Comment:
add the organization type as well. Going forward, when using the `hasRequiredScopes` use the organization type as well.

Check other places as well.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5161/commits/7db658aaf478c3c9f49ab169e0c2acb33e2d686c